tests/auth_tests/test_forms.py

```diff
@@ -872,3 +872,118 @@ def test_password_whitespace_not_stripped(self):
         self.assertTrue(form.is_valid())
         self.assertEqual(form.cleaned_data['password1'], data['password1'])
         self.assertEqual(form.cleaned_data['password2'], data['password2'])
+
+
+@override_settings(AUTH_PASSWORD_VALIDATORS=[
```

Comment:
I think it's enough to include a single test in `UserCreationFormTest`. Testing for both first_name and last_name seem unnecessary. 

Adding tests for PasswordChangeForm that aren't affected by this change shouldn't be done in this commit.